app/main.py

```diff
@@ -1,8 +1,63 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ) -> None:
+        if not (1 <= comfort_class <= 7):
+            print("Error")
+        if not (1 <= clean_mark <= 10):
+            print("Error")
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: float,
+        clean_power: bool,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        if not (1.0 <= distance_from_city_center <= 10.0):
+            print("Error")
+        if not (1 <= clean_power <= 10):
+            print("Error")
+        if not (1.0 <= average_rating <= 5.0):
+            print("Error")
```

Comment:
What if I add this code at the end?
```
wrong_car = Car(-100, -100, "name")
Car.serve_cars([wrong_car])
```
It'll print this text, but the program will continue to work with incorrect data, thus you should either set correct data or stop program